server/etcdserver/apply/uber_applier.go

```diff
@@ -130,19 +132,29 @@ func (a *uberApplier) dispatch(r *pb.InternalRaftRequest, shouldApplyV3 membersh
 	switch {
 	case r.Range != nil:
 		op = "Range"
-		ar.Resp, ar.Trace, ar.Err = a.applyV3.Range(r.Range)
+		ctx := context.Background()
+		ar.Resp, ar.Err = a.applyV3.Range(ctx, r.Range)
+		ar.Trace = traceutil.Get(ctx)
```

Comment:
This context will not have trace attached. Attaching a value to context creates a sub context and doesn't affect the  original context.